gateway/webhook/webhook.go

```diff
@@ -265,9 +265,9 @@ func (webhook *HandleT) Register(name string) {
 }
 
 func newWebhookStat(destID string) *webhookSourceStatT {
-	numEvents := stats.NewDestStat("webhook_num_events", stats.CountType, destID)
-	numOutputEvents := stats.NewDestStat("webhook_num_output_events", stats.CountType, destID)
-	sourceTransform := stats.NewDestStat("webhook_dest_transform", stats.TimerType, destID)
+	numEvents := stats.NewTaggedStat("webhook_num_events", stats.CountType, map[string]string{"destID": destID,})
```

Comment:
Maybe adding a type in stats package for Tags is better than repeating map[string]string everywhere?
```type Tags map[string]string```
and use it as 
```..., stats.Tags{"destID": destID,}))```

Reply:
Created a new type. Used it instead of map every where in context of TaggedStat.
Done